src/main/java/nl/cherement/jak/entity/UserEntity.java

```diff
@@ -12,81 +11,40 @@
 @Entity
 @Table(name = "users")
 public class UserEntity implements Serializable {
+
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    private long id;
+    public long id;
 
     @Column(nullable = false, unique = true)
-    private String username;
+    public String username;
 
     @Column(nullable = false)
     @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
-    protected String password;
-
-    private int active;
-    private String roles = "";
-    private String permissions = "";
-
-    public long getId() {
-        return id;
-    }
-
-    public void setId(long id) {
-        this.id = id;
-    }
+    public String password;
 
-    public void setUsername(String username) {
-        this.username = username;
-    }
-
-    public void setPassword(String password) {
+    @Column(name="active")
+    public boolean active;
```

Comment:
`isEnabled()` in `UserPrincipal` moet hier ook recht mee gezet worden. 
Deze gaat nu uit van een int.